### DISCUSSION TITLE
Welcome to QBJS Discussions!

### DISCUSSION THREAD

**Discussion by boxgaming** [boxgaming/qbjs#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by felixp7** [comment]
Hi there! Thanks for setting this up. I wrote a whole bunch of games in Basic. Mostly sdlBasic, that worked best for me. Also wrote a book about Basic, and in fact just revived one of my old interpreters.

To answer a question you asked elsewhere, I'd like to help you write a README that's more than literally just five words. Could simply submit a pull request if that's all right.

**Reply by boxgaming** [reply]
Welcome @felixplesoianu!  Yes, I would welcome the README submission!  Pull request sounds good.
